source/Planet.cpp

```diff
@@ -574,7 +575,7 @@ string Planet::DemandTribute(PlayerInfo &player) const
 // While being tributed, attempt to spawn the next specified defense fleet.
 void Planet::DeployDefense(list<shared_ptr<Ship>> &ships) const
 {
-	if(!isDefending || Random::Int(60) || defenseDeployed == defenseFleets.size())
+	if(!isDefending || Random::Int(FRAME_RATE) || defenseDeployed == defenseFleets.size())
```

Comment:
Here again it is just a coincidence that we currently launch about 1 defense fleet per second.